### PR TITLE
Replace NSLog with print

### PR DESCRIPTION
**Pull request by aeromusek**
@derrickstaten can you thumb this one too?

I only updated the Swift example...I couldn't tell from reading whether NSLog is still the recommended option for Objective-C.

### CODE REVIEW

File: pages/getting-started/creating-links-in-apps.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ Finally, generate the link by referencing the `BranchUniversalObject` you create
 {% highlight swift %}
 branchUniversalObject.getShortUrlWithLinkProperties(linkProperties,  andCallback: { (url: String?, error: NSError?) -> Void in
     if error == nil {
-        NSLog("got my Branch link to share: %@", url)
+        print("got my Branch link to share: %@", url)
```
Comment (derrickstaten) [inline pages/getting-started/creating-links-in-apps.md]: i think it should actually be `print("got my Branch link to share:", url)`, can you double check sometime? you could just add a print statement inside `application:didFinishLaunchingWithOptions:`

**Comment by dwestgate** [comment]
👍

File: pages/getting-started/creating-links-in-apps.md (hunk 1 above)
Comment (derrickstaten) [inline pages/getting-started/creating-links-in-apps.md]: please see this comment before merging. we should invest 2 minutes into testing this before merging

**Comment by aeromusek** [comment]
See #285

**Pull request closed without merging**